storage/mysql/admin_storage.go

```diff
@@ -301,10 +293,12 @@ func (t *adminTX) CreateTree(ctx context.Context, tree *trillian.Tree) (*trillia
 	}
 
 	now := time.Now()
-	nowDatetime := toDatetime(now)
+	nowMillis := toMillisSinceEpoch(now)
```

Comment:
nit: inline `time.Now()` here? We don't seem to need additional formats anymore.

Ditto @ line 405 (UpdateTree).